app/rpc.js

```diff
@@ -37,6 +37,18 @@
         "thread-open": async function(threadId) {
             await F.mainView.openThreadById(threadId);
         },
+         
+        "thread-call-start": async function(threadId, autoJoinBool) {
+            const thread = requireThread(threadId);
+            const callMgr = F.calling.getOrCreateManager(thread.id, thread);
+            await callMgr.start({autoJoin: autoJoinBool});
+        },
+ 
```

Comment:
Couple things here:

The callmanager technically wants a callId (not threadId) and a thread argument.  They can and often are the same, but the way the protocol works, they are permitted to be different.  Some clients may find this useful if they want to devise their own sort of calling features that are multitenant.  Eg. multiple calls per thread.

Instead of a bool argument, Let's just have them pass in an options object that is optional.  This pattern allows the rpc call to be open ended and we don't have to change it for every option we want to pass in later on.  a la.,
```javascript
       "thread-call-start": async function(callId, options) {
            options = options || {};
            threadId = options.threadId || callId;
￼            const thread = requireThread(threadId);
￼            const callMgr = F.calling.getOrCreateManager(callId, thread);
￼            await callMgr.start(options);
￼        },
```

So this could be used like `thread-call-start(somethreadId, {autoJoin: true})`;  (or without the options object).  Or if they are advanced they can override the threadId in the options.